public/js/edit.js

```diff
@@ -174,6 +174,15 @@ Edit.getTags = function () {
                 });
             }
 
+            if (result.data.summary && result.data.summary !=="") {
+                $("#summary").val(result.data.summary);
+                LRR.toast({
+                    heading: "Archive summary changed to :",
+                    text: result.data.summary,
```

Comment:
Since summaries can be pretty long it's probably better to just mention "Archive summary updated!" without the text.  

Mentioning the value is kinda superflous already since users can just like, see it in the text box..